src/components/index.ts

```diff
@@ -4,3 +4,4 @@ export { Header } from "./header/Header"
 export { LinkButton } from "./linkButton/LinkButton"
 export { SplashText } from "./splashText/SplashText"
 export { TextBlock } from "./textBlock/TextBlock"
+export { CodeOfConductText } from "./codeOfConductText/CodeOfConductText"
```

Comment:
Let's keep these imports in alphabetical order

Reply:
👍